### DISCUSSION TITLE
[REQUEST THREAD] What examples would you like to be added?

### DISCUSSION THREAD

**Discussion by irishgreencitrus** [irishgreencitrus/raylib.v#14] [Ideas]
As title.
Link examples you would like to see implemented in raylib.v

**Comment by timi5691** [comment]
add example https://www.raylib.com/examples/textures/loader.html?name=textures_mouse_painting

**Comment by irishgreencitrus** [comment]
> add example https://www.raylib.com/examples/textures/loader.html?name=textures_mouse_painting

Implemented in cc95b68
